diff --git a/dist/node/chunks/config.js b/dist/node/chunks/config.js
index 2b39a20c497ee5c9f1cefdf1781df9955b50b97d..ccdda2dfe26a337bf8052cb47d409288453fbf4b 100644
--- a/dist/node/chunks/config.js
+++ b/dist/node/chunks/config.js
@@ -28793,6 +28793,7 @@ var EnvironmentPluginContainer = class {
 			const start = debugPluginTransform ? performance$1.now() : 0;
 			let result;
 			const handler = getHookHandler(plugin.transform);
+			if (!handler) continue;
 			try {
 				result = await this.handleHookPromise(handler.call(ctx, code, id, optionsWithSSR));
 			} catch (e$1) {
